### PR TITLE
feat(azure-db): add azure-db container

### PR DESCRIPTION
**Pull request by revolunet**
Add a new azure-db container to help with managed databases

- create db and user
- destroy db and user
- backup db on azure blob
- Handle hasura perms

https://docs.microsoft.com/fr-fr/azure/postgresql/concepts-servers#managing-your-server

> L’attribut de superutilisateur PostgreSQL est affecté à azure_superuser, qui appartient au service managé. Vous n’avez pas accès à ce rôle.

### CODE REVIEW

File: azure-db/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ARG PG_IMAGE=postgres:11-alpine
+
+FROM ${PG_IMAGE}
```
Comment (douglasduteil) [inline azure-db/Dockerfile]: Why `postgres:11-alpine` is a variable ?

File: azure-db/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+RUN apk add --no-cache python python-dev build-base gcc wget git py-pip py-setuptools libffi-dev openssl-dev
+RUN pip install azure-cli
+RUN apk del python-dev build-base gcc wget py-pip libffi-dev openssl-dev
```
Comment (douglasduteil) [inline azure-db/Dockerfile]: Can merge the runs for better docker caching

File: azure-db/Dockerfile (hunk 1 above)
Comment (revolunet) [reply azure-db/Dockerfile]: unlikely but we could build the images for different PG versions.

File: azure-db/Dockerfile (hunk 2 above)
Comment (revolunet) [reply azure-db/Dockerfile]: merge with `&&`

Comment (revolunet) [reply azure-db/Dockerfile]: ?

**Comment by revolunet** [comment]
I could add test, but end-to-end, which need an Azure PG

File: azure-db/Dockerfile (hunk 1 above)
Comment (revolunet) [reply azure-db/Dockerfile]: (important for backups)

File: azure-db/Dockerfile (hunk 2 above)
Comment (revolunet) [reply azure-db/Dockerfile]: do the fix :)

**Comment by revolunet** [comment]
up :)

**Comment by douglasduteil** [comment]
I need help faking the `azure_pg_admin` role.
I did not succeed trying to connect to an azura instance.
Let's look at that tomorrow ;)

**Comment by revolunet** [comment]
k thanks !

**Pull request merged**